xml/tuning-architecture.xml

```diff
@@ -0,0 +1,338 @@
+       a &ceph; cluster network:
+     </para>
+     <itemizedlist>
+       <listitem>
+         <para>
+           Utilize 25/50/100 GbE connections - The signaling rate is 2.5
+           times faster than 10/40GbE resulting in lower latency over
+           the wire. The impact from the faster signaling rate would be
+           minimal with spinning media and more impactful with faster
+           technologies like NVMe.
+         </para>
+       </listitem>
+       <listitem>
+         <para>
+           Network bandwidth should be at least the total bandwidth of all
+           storage devices present in the storage node
+         </para>
+       </listitem>
+       <listitem>
+         <para>
+           Usage of VLANs on LACP bonded ethernet provides the best
+           balance of bandwidth aggregation + fault tolerance
+         </para>
+       </listitem>
+       <listitem>
+         <para>
+           The network should utilize jumbo frame ethernet if all nodes
+           connecting to the storage are able to do so, otherwise use the
+           standard MTU
+         </para>
+       </listitem>
+     </itemizedlist>
+   </sect2>
+ </sect1>
+ <sect1 xml:id="tuning-node-hw-recommendations">
+   <title>Node Hardware Recommendations</title>
+   <sect2>
+     <title>CPU</title>
+     <para>
+       There are a lot of options when it comes to processor selection in
+       the market today. There are multiple varieties available from the
+       major vendors in the x86 space and a wide array of options in the
+       64-bit ARM space as well. Without regard to the core architecture,
+       there is one rule that is always true, and that is that higher
+       clockspeed allows more work to be done in the same amount of time.
+       This consideration is most important when working with higher speed
+       storage and network devices.
+     </para>
+     <para>
+       CPU selection is also an important consideration for specific services.
+       Some services, such as metadata servers, NFS, Samba, and ISCSI
+       gateways benefit from a smaller number of much faster cores, while
+       the OSD nodes need a more core dense solution.
+     </para>
+     <para>
+       A second consideration is whether to use a single socket or multiple
+       sockets. The answer to this will depend on the device density,
+       type of network hardware being utilized, etc. In many nodes, a single
+       socket will provide better performance as the processor interlink is
+       a bottleneck, though this would most likely be noticed in an all
+       NVMe based node type. The general recommendation is to use a single
+       socket whenever possible.
+     </para>
+     <para>
+       When considering which processor to choose, there are several
+       considerations aside from clock-speed that should be taken into
+       consideration. They include:
+     </para>
+     <itemizedlist>
```

Comment:
I suggest a variable list for this, since each entry contains a term then an explanation. Let DAPS do the formatting legwork.

Reply:
Now, I think there's a reason I didn't do that for this list... I'll check again :)

Reply:
So, I didn't change this because of the original formatting. I'm going to leave it how it is, but I'll keep an note.